tsconfig.json

```diff
@@ -21,6 +21,8 @@
         "out": "docs/build",
         "name": "parse5",
         "readme": "README.md",
+        "emit": "docs",
```

Comment:
Note: This is the default

Reply:
Ah, I didn't realise — thanks for flagging.

This PR fixed our builds by the way, as can be seen in https://github.com/inikulin/parse5/commits/gh-pages. One way I was able to replicate the issue locally is to reduce the memory limit of Node using `--max-old-space-size=1024`.